src/View/Music/listAll.html.twig

```diff
@@ -0,0 +1,31 @@
+{% extends 'layout.html.twig' %}
+{% block content %}
+    {% include 'Music/_button_genre.html.twig' %}
+	<main class="container">
+		<div class="row justify-content-around">
+			{% for musical in musics %}
+				{% if musical.genre_name == 'pop' %}
+				{% set class_genre = 'card-pop'%}
+				{% elseif musical.genre_name == 'cinematic' %}
+				{% set class_genre = 'card-cinematic'%}
+				{% elseif musical.genre_name == 'electro' %}
+				{% set class_genre = 'card-electro'%}
+				{% elseif musical.genre_name == 'world' %}
+				{% set class_genre = 'card-world'%}
+				{% elseif musical.genre_name == 'chillout' %}
+				{% set class_genre = 'card-chillout'%}
+				{% elseif musical.genre_name == 'ambient' %}
+				{% set class_genre = 'card-ambient'%}
+				{% endif %}
+				<div class="{{ class_genre }} mb-5" style="width: 20rem">
```

Comment:
This could be very simplier if you withdraw the if/set assemblage and replace it in your div with something like that :
`<div class="card-{{ musical.genre_name }} mb-5" style="width: 20rem">`

Reply:
Yes, thank you it's more simple, i don't know i can tell the card class like this